yt_dlp/extractor/voicy.py

```diff
@@ -0,0 +1,152 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import compat_str
+from ..utils import (
+    ExtractorError,
+    smuggle_url,
+    traverse_obj,
+    unsmuggle_url,
+    unified_strdate,
+)
+
+import re
+import itertools
+
+
+class VoicyBaseIE(InfoExtractor):
+    # every queries are assumed to be a playlist
+    def _extract_from_playlist_data(self, value):
+        voice_id = compat_str(value.get('PlaylistId'))
+        upload_date = unified_strdate(value.get('Published'), False)
+        items = [self._extract_single_article(voice_data) for voice_data in value['VoiceData']]
+        return {
+            '_type': 'playlist',
```

Comment:
I thought you were gonna make this `multi_video`?

Reply:
oh, I have forgotten about that